src/import/FSHImporter.ts

```diff
@@ -1916,6 +1961,70 @@ export class FSHImporter extends FSHVisitor {
     }
   }
 
+  /**
+   * Given a path and the context containing it, apply the path context indicated by the path's indent
+   * @param path - The path to apply context to
+   * @param parentCtx - The parent element containing the path
+   * @returns {string} - The path with context prepended
+   */
+  private prependPathContext(path: string, parentCtx: any): string {
+    const location = this.extractStartStop(parentCtx);
+    const currentIndent = location.startColumn - this.baseIndent;
+
+    if (currentIndent > 0 && this.pathContext.length === 0) {
+      logger.error(
+        'The first rule of a definition cannot be indented. The rule will be processed as if it is not indented.',
+        location
```

Comment:
I've done this on this PR too many times, but I thought of something else after I approved. I just realized that it doesn't seem like these locations are logged with the errors. Are you seeing that as well?

Reply:
Hm yeah I think these actually all need to be updated so they fit the `SourceInfo` type. I think doing something like this should fix it:

```
{ location, file: this.currentFile }
```

I think this will really help some of those projects that just have new error logs about indents but nothing actually changes in the output.

Reply:
https://github.com/FHIR/sushi/pull/797/commits/dcd921223931f70f590ea393e9096bf86623cef8 should fix this.